test/api/v01/output_test.rb

```diff
@@ -265,21 +265,21 @@ def test_returned_keys_json_result
         problem: VRP.lat_lon,
         solver_name: 'vroom',
         expected_route_keys: %w[vehicle_id original_vehicle_id activities total_travel_time total_distance total_time start_time end_time],
-        expected_activities_keys: %w[point_id travel_distance travel_time travel_value service_id original_service_id detail current_distance type],
+        expected_activities_keys: %w[point_id travel_distance travel_time travel_value begin_time end_time service_id original_service_id detail current_distance type departure_time],
         expected_unassigned_keys: %w[point_id service_id original_service_id detail type reason]
       },
       ortools: {
         problem: VRP.lat_lon_capacitated,
-        solver_name: 'ortools',
-        expected_route_keys: %w[vehicle_id original_vehicle_id activities total_travel_time total_distance total_time total_waiting_time start_time end_time costs initial_loads],
-        expected_activities_keys: %w[point_id travel_distance travel_time travel_value waiting_time begin_time end_time service_id original_service_id pickup_shipment_id delivery_shipment_id original_shipment_id detail current_distance alternative type departure_time],
-        expected_unassigned_keys: %w[point_id service_id original_service_id shipment_id original_shipment_id detail type reason]
+        solver_name: 'vroom',
+        expected_route_keys: %w[vehicle_id original_vehicle_id activities total_travel_time total_distance total_time start_time end_time],
+        expected_activities_keys: %w[point_id travel_distance travel_time travel_value begin_time end_time service_id original_service_id pickup_shipment_id delivery_shipment_id original_shipment_id detail current_distance type departure_time],
+        expected_unassigned_keys: %w[point_id service_id original_service_id detail type reason]
       },
       periodic_ortools: {
         problem: VRP.lat_lon_two_vehicles,
-        solver_name: 'ortools',
-        expected_route_keys: %w[vehicle_id original_vehicle_id activities total_travel_time total_distance total_time total_waiting_time start_time end_time costs initial_loads],
-        expected_activities_keys: %w[point_id travel_distance travel_time travel_value waiting_time begin_time end_time departure_time service_id original_service_id detail current_distance alternative type],
+        solver_name: 'vroom',
```

Comment:
Is VROOM really able to solve scheduling problems when we do not call periodic heuristic? (i.e. problems with more than one visit per service)